frontend/src/components/advanced-auto-filter/advanced-auto-filter.tsx

```diff
@@ -236,6 +243,19 @@ const AdvancedAutoFilter: FC = () => {
         <UTurnRightIcon className={styles.resetIcon} />
         Reset All Filters
       </h6>
+      {isButtonVisible && (
+        <Zoom in={isButtonVisible}>
```

Comment:
`isButtonVisible &&` will make the `Zoom` immediately unmounted and the out animation won't work